modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -340,17 +574,53 @@ Type::
 |===
 | Property | Type | Description
 
-| `containerName`
+| `file`
 | `string`
-| `containerName` defines the name of the container to configure for IPFIX.
+| File name within the config map or secret
 
-| `daemonSetName`
+| `name`
 | `string`
-| `daemonSetName` defines the name of the DaemonSet controlling the OVN-Kubernetes pods.
+| Name of the config map or secret containing the file
 
 | `namespace`
 | `string`
-| Namespace where OVN-Kubernetes pods are deployed.
+| Namespace of the config map or secret containing the file. If omitted, the default is to use the same namespace as where Network Observability is deployed.
```

Comment:
Yeah I do think maybe the `Operator` bit needs to be here to make the capitalization more legit.

Reply:
So here this isn't the Operator on purpose, as the Operator could be in a different namespace than the other netobserv workloads.
Note that upstream we write "NetObserv" here and that's changed downstream to `Network Observability` because that's the product name

Reply:
note, if that makes it better, we could change this to:
```
... the default is to use the same namespace as defined in `spec.namespace`.
```
This is another way to say the same

Reply:
Tbh @jotak I think its OK as is. Thanks for providing the background reasoning here.